python/examples/advanced_agents/sales_kit/CRM_agent/main.py

```
@@ -0,0 +1,83 @@
+from composio_llamaindex import ComposioToolSet, Action
+from llama_index.core.agent import FunctionCallingAgentWorker
+from llama_index.core.llms import ChatMessage
+from llama_index.llms.openai import OpenAI
+from dotenv import load_dotenv
+from pathlib import Path
+import os
+import streamlit as st
+
+# Load environment variables
+load_dotenv()
+
+# Choose your LLM here
+llm = OpenAI(model='gpt-4o')
+# llm = Groq(model="llama3-groq-70b-8192-tool-use-preview")
+# llm = Cerebras(model="llama3.1-70b")
+
+# Initialize tools and apps
+composio_toolset = ComposioToolSet()
```

Consider adding error handling for the ComposioToolSet initialization. If API keys are missing or invalid, this could fail silently. Suggestion:

```python
try:
    composio_toolset = ComposioToolSet()
except Exception as e:
    st.error(f"Failed to initialize CRM tools: {str(e)}")
    sys.exit(1)
```